hw/top_earlgrey/cdc/cdc_waivers.tcl

```diff
@@ -96,3 +221,44 @@ set_rule_status -rule {W_DATA} -status {Waived} \
 set_rule_status -rule {S_GENCLK} -status {Waived} \
   -expression {(ClockTreeSignal=~"u_ast.u_ast_clks_byp.*") && (DrivenFlop =~ "u_ast.u_ast_clks_byp.*")} \
   -comment {ast clks are assumed to be checked by the partner as standalone.}
+
+set_rule_status -rule {S_GENCLK} -status {Waived} \
+  -expression {(ClockTreeSignal=~"IOC*") && (DrivenFlop =~ "u_ast.u_ast_clks_byp.*")} \
+  -comment {ast clks are assumed to be checked by the partner as standalone.}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived} \
+  -expression {(DrivingSignal=~"*.u_rstmgr_aon.*.u_rst_sync*.u_impl_generic.q_o*") && (ReceivingFlop =~ "*u_rstmgr_aon.*.gen_rst_chk.u_rst_chk.child_rst_asserted*")} \
+  -comment {asynchronous reset generation in rstmgr}
+
+set_rule_status -rule {W_RECON_GROUPS} -status {Waived} -expression {(ControlSignal=~"*u_sync_1.gen_generic.u_impl_generic.q_o**") && (ReconSignal=~"*u_rv_core_ibex*.*rdata*")} -comment {multiple sources with mux}
```

Comment:
@jeoongp when you say multiple sources with mux, do you mean the mux select helps resolve the reconvergence issue? Just that the tool does not understand?

Reply:
Yes, I think so. The feedback path is staged by a FF. But, I am not sure of why the staged signal is detected as a reconvergence path as follows.
<img width="733" alt="Screen Shot 2022-08-24 at 10 01 35 AM" src="https://user-images.githubusercontent.com/103145460/186479387-f981182b-d0d4-4b6e-827d-426dc1199c48.png">